web/cypress/integration/route/batch-delete-route.spec.js

```diff
@@ -0,0 +1,114 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable */
+
+context('Create and Batch Deletion Routes', () => {
+  const timeout = 5000;
+
+  const selector = {
+    name: '#name',
+    description: '#desc',
+    hosts_0: '#hosts_0',
+    uris_0: '#uris_0',
+    labels_0_labelKey: '#labels_0_labelKey',
+    labels_0_labelValue: '#labels_0_labelValue',
+    nodes_0_host: '#submitNodes_0_host',
+    nodes_0_port: '#submitNodes_0_port',
+    nodes_0_weight: '#submitNodes_0_weight',
+    drawerBody: '.ant-drawer-wrapper-body',
+    notification: '.ant-notification-notice-message',
+    notificationClose: '.anticon-close',
+  };
+
+  const data = {
+    host1: '11.11.11.11',
+    host2: '12.12.12.12',
+    port: '80',
+    weight: 1,
+    uris: '/get',
+    uris0: '/get0',
+    uris1: '/get1',
+    uris2: '/get2',
+    urisx: '/getx',
+    submitSuccess: 'Submit Successfully',
+    deleteRouteSuccess: 'Delete Route Successfully',
+    test: 'test',
+    test0: 'test0',
+    test1: 'test1',
+    test2: 'test2',
+    testx: 'testx',
+    value0: 'value0',
+    label0_value0: 'label0:value0',
+  };
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('should create route test0, test1, test2', function () {
+    cy.visit('/');
+    cy.contains('Route').click();
+    for (let i = 0; i < 3; i += 1) {
+      cy.contains('Create').click();
+      cy.contains('Next').click().click();
+      cy.get(selector.name).type(`test${i}`);
+      cy.get(selector.description).type(`desc${i}`);
+      cy.get(selector.hosts_0).type(data.host1);
+      cy.get(selector.uris_0).clear().type(`/get${i}`);
+
+      // config label
+      cy.contains('Manage').click();
+
+      // eslint-disable-next-line no-loop-func
+      cy.get(selector.drawerBody).within(() => {
+        cy.contains('button', 'Add')
+          .should('not.be.disabled')
+          .click()
+          .then(() => {
+            cy.get(selector.labels_0_labelKey).type(`label${i}`);
+            cy.get(selector.labels_0_labelValue).type(`value${i}`);
+            cy.contains('Confirm').click();
+          });
+      });
+
+      cy.contains('button', 'Next').should('not.be.disabled').click();
+      cy.get(selector.nodes_0_host).type(data.host2, {
+        timeout,
+      });
+      cy.get(selector.nodes_0_port).type(data.port);
+      cy.get(selector.nodes_0_weight).type(data.weight);
+      cy.contains('Next').click();
+      cy.contains('Next').click();
+      cy.contains('Submit').click();
+      cy.contains(data.submitSuccess);
+      cy.contains('Goto List').click();
+      cy.url().should('contains', 'routes/list');
+    }
```

Comment:
It is best to assert that 3 corresponding routes were successfully created

Reply:
ok